File: core/src/com/alta_v2/rendering/utils/GradientResource.java
package com.alta_v2.rendering.utils;

import com.badlogic.gdx.graphics.Color;
import com.google.common.collect.Lists;
import lombok.experimental.UtilityClass;

import java.util.HashMap;
import java.util.List;
import java.util.Map;

@UtilityClass
public class GradientResource {

    public Map<Gradient, List<Color>> gradients = new HashMap<Gradient, List<Color>>() {
        {
            put(Gradient.ORANGE, Lists.newArrayList(new Color(0xFF7400AA), new Color(0xA64B00BB), new Color(0xA64B00AA), new Color(0xFF9640BB)));
        }
    };

    public enum Gradient {
        ORANGE
    }

}

File: core/src/com/alta_v2/rendering/common/utils/GradientResource.kt
package com.alta_v2.rendering.common.utils

import com.badlogic.gdx.graphics.Color

enum class Gradient {
    ORANGE
}

val gradients = mapOf<Gradient, List<Color>>(
        Gradient.ORANGE to listOf(Color(-0x8bff56), Color(-0x59b4ff45), Color(-0x59b4ff56), Color(-0x69bf45))
)